tests/unit/llm/test_anthropic.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from unittest.mock import AsyncMock, MagicMock, Mock, patch
+
+import nest_asyncio
+import pytest
+from distilabel.llm.anthropic import AnthropicLLM
+
+
+@patch("anthropic.AsyncAnthropic")
+class TestAnthropicLLM:
+    def test_anthropic_llm(self, mock_anthropic: MagicMock) -> None:
```

Comment:
```suggestion
    def test_anthropic_llm(self) -> None:
```

Reply:
Hi here @sdiazlor this feedback was wrong, I think you need to explicitly provide it, my bad!